src/util/DefaultValueProcessor.ts

```diff
@@ -129,6 +144,42 @@ const addingDefaultProcedureProperties = (
   return procedureEntry;
 };
 
+const addConditionCategory = (
+  conditionEntry: Condition,
+  conditionReferences: string[]
+) => {
+  if (!conditionEntry?.category) {
```

Comment:
Be careful here, `!conditionEntry?.category` evaluates to `true` when `conditionEntry` is null or undefined.  That would cause an exception on the next line when trying to access `conditionEntry.id`

Reply:
Fixed in next push